proto/mcu_pb.proto

```diff
@@ -158,6 +158,10 @@ enum LogEventCode {
   backend_started = 141;
   mcu_shutdown = 142;
   backend_shutdown = 143;
+  sfm3019_air_disconnected = 144;
+  sfm3019_o2_disconnected = 145;
+  fdo2_disconnected = 146;
+  critical_battery = 147;
```

Comment:
The battery alarm codes should be next to each other, and their names should be consistent (e.g. `battery_low` and `battery_critical` have consecutive values such as 138 and 139). Feel free to renumber the enum values as needed.

Reply:
yep! was thinking whether you wanted them in that order.